src/test/java/com/commercetools/sync/sdk2/categories/CategorySyncMockUtils.java

```diff
@@ -0,0 +1,236 @@
+    final CategoryDraft categoryDraft = mock(CategoryDraft.class);
+    when(categoryDraft.getName()).thenReturn(LocalizedString.of(locale, name));
+    when(categoryDraft.getSlug()).thenReturn(LocalizedString.of(locale, slug));
+    when(categoryDraft.getKey()).thenReturn(key);
+    when(categoryDraft.getExternalId()).thenReturn(externalId);
+    when(categoryDraft.getDescription()).thenReturn(LocalizedString.of(locale, description));
+    when(categoryDraft.getMetaDescription())
+        .thenReturn(LocalizedString.of(locale, metaDescription));
+    when(categoryDraft.getMetaTitle()).thenReturn(LocalizedString.of(locale, metaTitle));
+    when(categoryDraft.getMetaKeywords()).thenReturn(LocalizedString.of(locale, metaKeywords));
+    when(categoryDraft.getOrderHint()).thenReturn(orderHint);
+    when(categoryDraft.getParent())
+        .thenReturn(CategoryResourceIdentifierBuilder.of().id(parentId).build());
+    return categoryDraft;
+  }
+
+  public static CategoryDraft getMockCategoryDraft(
+      @Nonnull final Locale locale,
+      @Nonnull final String name,
+      @Nonnull final String key,
+      @Nullable final String parentKey,
+      @Nonnull final String customTypeKey,
+      @Nonnull final Map<String, Object> customFields) {
+    return getMockCategoryDraftBuilder(locale, name, key, parentKey, customTypeKey, customFields)
+        .build();
+  }
+
+  /**
+   * Given a {@code locale}, {@code name}, {@code slug} and {@code key}; this method creates a mock
+   * of {@link CategoryDraft} with all those supplied fields. All the supplied arguments are given
+   * as {@link String} and the method internally converts them to their required types. For example,
+   * for all the fields that require a {@link LocalizedString} as a value type; the method creates
+   * an instance of a {@link LocalizedString} with the given {@link String} and {@link Locale}.
+   *
+   * @param locale the locale to create with all the {@link LocalizedString} instances.
+   * @param name the name of the category.
+   * @param slug the slug of the category.
+   * @param key the key of the category.
+   * @return an instance {@link CategoryDraft} with all the given fields set in the given {@link
+   *     Locale}.
+   */
+  public static CategoryDraft getMockCategoryDraft(
+      @Nonnull final Locale locale,
+      @Nonnull final String name,
+      @Nonnull final String slug,
+      @Nullable final String key) {
+    final CategoryDraft mockCategoryDraft = mock(CategoryDraft.class);
+    when(mockCategoryDraft.getName()).thenReturn(LocalizedString.of(locale, name));
+    when(mockCategoryDraft.getSlug()).thenReturn(LocalizedString.of(locale, slug));
+    when(mockCategoryDraft.getKey()).thenReturn(key);
+    when(mockCategoryDraft.getCustom()).thenReturn(getMockCustomFieldsDraft());
+    return mockCategoryDraft;
+  }
+
+  /**
+   * Given a {@code locale}, {@code name}, {@code slug}, {@code key}, {@code description}, {@code
```

Comment:
I think these comments are not useful at all. This is a test class so it's probably used by us and to read the comments is much harder to understand than just to look at the code 😅